sql-mutations/diplomat.sql

```diff
@@ -0,0 +1,5 @@
+select * from addresses where "addressId" = 161;
```

Comment:
This isn't really necessary. If you want to output the address that you just deleted you could use `returning *` at the end of the `delete` statement.